Add note about crashes in old android devices due to Billing client issue (#713)
RevenueCat/docs: docs/revenuecat-support/general-troubleshooting.mdx

If your app is being rejected from the respective store it can be for many different reasons. We have a useful document [here](https://www.revenuecat.com/docs/test-and-launch/app-store-rejections) that goes over common reasons for rejection and how to help get your app approved!

> ### Seeing crashes like `NoClassDefFoundError: com.android.billingclient.api.zzce` in your android app?

Author: Added this to the general troubleshooting section. Thought about adding to the Android SDK section, but since this can happen in the hybrids as well, it didn't feel right. Lmk what you think!

If you're seeing a crash similar to:

```
java.lang.NoClassDefFoundError: com.android.billingclient.api.zzce

> 	at com.android.billingclient.api.BillingFlowParams$Builder.build(com.android.billingclient:billing@@7.1.1:3)

Reviewer: Maybe this line (and the section header) should also mention our own exception type? Otherwise people that get sent here from our own exception might be confused?

```

> This issue was introduced by Google in the Billing client library in version 7.1.+. It has been reported [here](https://issuetracker.google.com/issues/377466571) and [here](https://issuetracker.google.com/issues/404645683).

Reviewer: ```suggestion
This issue was introduced by Google in the Billing client library in version 7.1.+. It has been reported to Google [here](https://issuetracker.google.com/issues/377466571) and [here](https://issuetracker.google.com/issues/404645683).
```

The Play Billing Library started using certain Java 8 types, which not all devices can handle. Concretely this issue may happen in specific older Android models.

> Until they fix this, the current workaround is to enable Core Library Desugaring in your Android app. To do so, you can follow the instructions [here](https://developer.android.com/studio/write/java8-support#library-desugaring). 

Reviewer: ```suggestion
Until Google fixes this, the current workaround is to enable Core Library Desugaring in your Android app. To do so, you can follow the instructions [here](https://developer.android.com/studio/write/java8-support#library-desugaring). 
```

If your app is being rejected from the respective store it can be for many different reasons. We have a useful document [here](https://www.revenuecat.com/docs/test-and-launch/app-store-rejections) that goes over common reasons for rejection and how to help get your app approved!

> ### Seeing crashes like `NoCoreLibraryDesugaringException` or `NoClassDefFoundError: com.android.billingclient.api.zzce` in your android app?

Reviewer: ```suggestion
### Seeing crashes like `NoCoreLibraryDesugaringException` or `NoClassDefFoundError: com.android.billingclient.api.zzce` in your Android app?
```
